lib/uiautomator2.js

```diff
@@ -116,15 +161,12 @@ class UiAutomator2Server {
     );
 
     this.log.debug(`Server packages status: ${JSON.stringify(packagesInfo)}`);
-    // Enforce server packages reinstall if any of the packages is not installed, while the other is
-    const shouldUninstallServerPackages = (packagesInfo.some(({installState}) => installState === this.adb.APP_INSTALL_STATE.NOT_INSTALLED)
-          && !packagesInfo.every(({installState}) => installState === this.adb.APP_INSTALL_STATE.NOT_INSTALLED));
+    // To ensure if the UIA2 driver should uninstall UI2 server packages from the device
```

Comment:
This comment is now not really needed since we have the method itself properly documented